sjb/config/common/test_cases/kubernetes.yml

```diff
@@ -0,0 +1,110 @@
+---
+parameters:
+- name: buildId
+  description: "Unique build number for each run."
+- name: PULL_BASE_REF
+  description: "Ref name of the base branch."
+- name: PULL_BASE_SHA
+  description: "Git SHA of the base branch."
+- name: PULL_REFS
+  description: "All refs to test."
+- name: PULL_NUMBER
+  description: "Pull request number."
+- name: PULL_PULL_SHA
+  description: "Pull request head SHA."
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+sync_repos:
+  - name: "aos-cd-jobs"
+actions:
+  - type: "script"
+    title: "clone k8s and check out the correct refs"
+    script: |-
+      mkdir -p src/k8s.io/kubernetes
+      cd src/k8s.io
+      git clone https://github.com/kubernetes/kubernetes.git
+      cd kubernetes
+      git checkout master
+      # Take the latest commit of the master HEAD and store it into a file
+      git rev-parse HEAD > PULL_BASE_SHA
+
+      # Get version for the started.json
+      export KUBE_ROOT=.
+      export KUBE_GIT_TREE_STATE="clean"
+      source hack/lib/version.sh
+      kube::version::get_version_vars
+      echo $KUBE_GIT_VERSION > KUBE_GIT_VERSION
+  - type: "host_script"
+    title: "Get kubernetes PULL_BASE_SHA"
+    script: |-
+      SLAVE_HOME=$(ssh -F ./.config/origin-ci-tool/inventory/.ssh_config -t openshiftdevel "bash -l -c \"echo \${HOME}\"")
+      scp -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel:${SLAVE_HOME}/src/k8s.io/kubernetes/PULL_BASE_SHA PULL_BASE_SHA
+      scp -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel:${SLAVE_HOME}/src/k8s.io/kubernetes/KUBE_GIT_VERSION KUBE_GIT_VERSION
+  - type: "script"
+    title: "record the starting metadata"
+    timeout: 300
+    repository: "aos-cd-jobs"
+    script: |-
+      trap 'exit 0' EXIT
+      sjb/gcs/started.py
```

Comment:
Did we need any fields from this over the minimalist approach for CRI-O? With the latter we at least would not need the JQ stuff since we are creating the file here

Reply:
I have not checked the job yet.

Reply:
Checked the job. Hard to say which fields are actually needed as there is no official doc that describes all of them and their meaning. @lsm5 are you familiar with what is the expected content of both ``started.json`` and ``finished.json``?

Reply:
He's using the minimal impl in the CRI-O tests. I would maybe suggest that as a starting place since it's a little simpler than the python stuff.

Reply:
Done